mPower2/mPower2/WithdrawalViewController.swift

```diff
@@ -86,55 +86,69 @@ class WithdrawalViewController: UIViewController {
     
     @objc
     func withdrawalTapped() {
-        guard let appDelegate = UIApplication.shared.delegate as? AppDelegate,
-        let consentStatuses = appDelegate.userSessionInfo?.consentStatuses as? [ String : SBBConsentStatus ]
+        guard let withdrawalSurvey : SBBSurveyReference = SBABridgeConfiguration.shared.allSurveys().first(where: { (ref) -> Bool in
+                return ref.identifier == .withdrawal
+            })
```

Comment:
FYI, You could also do this:
```
let withdrawalSurvey = SBABridgeConfiguration.shared.allSurveys().first (where: { $0.identifier == .withdrawal })
```

B/c the `allSurveys()` method returns `[SBBSurveyReference]`, you don't need to cast it.

Reply:
thanks